netkan/netkan/metadata.py

```diff
@@ -59,12 +59,19 @@ def hook_only(self):
             return False
         return self.on_spacedock
 
-    def sqs_message(self):
+    def sqs_message_attribs(self, ckan_group=None):
+        attribs = {}
+        if ckan_group and not getattr(self, 'x_netkan_allow_out_of_order', False):
+            attribs['HighestVersion'] = ckan_group.highest_version().string
+        return attribs
+
+    def sqs_message(self, ckan_group=None):
         return {
-            'Id': self.identifier,
-            'MessageBody': self.contents,
-            'MessageGroupId': '1',
+            'Id':                     self.identifier,
+            'MessageBody':            self.contents,
+            'MessageGroupId':         '1',
             'MessageDeduplicationId': uuid.uuid4().hex,
+            'MessageAttributes':      self.sqs_message_attribs(ckan_group),
         }
```

Comment:
PEP8 Nitpick -> https://www.python.org/dev/peps/pep-0008/#pet-peeves

Reply:
Do any of those apply to this case? Doesn't look like it:

> Immediately inside parentheses, brackets or braces.
> Between a trailing comma and a following close parenthesis.
> Immediately before a comma, semicolon, or colon:
> Immediately before the open parenthesis that starts the argument list of a function call:
> Immediately before the open parenthesis that starts an indexing or slicing:
> More than one space around an assignment (or other) operator to align it with another.

Reply:
This, even if it doesn't specifically mention dictionaries.
> More than one space around an assignment (or other) operator to align it with another.

Reply:
OK.